Publish du html a une liste de mots (partie 1)

marie-flesch opened the pull request:
I am publishing the lesson "Du HTML à une liste de mots (partie 1)

marie-flesch committed:
publish Du HTML à une liste de mots (partie 1)

adding lesson file

marie-flesch committed:
Update du-html-a-une-liste-de-mots-1.md

changing "translation reviewers" to "translation reviewer"

spapastamkou reviewed (changes requested):
faudrait peut-être descendre le niveau des titres de 1 à chaque fois: les titres niveau 1 => niveau 2 et ceux de niveau 2 => niveau 3 (pour harmoniser avec les autres versions de la leçon en EN, PT et ES). Sinon cela a l'air top!

marie-flesch commented:
> faudrait peut-être descendre le niveau des titres de 1 à chaque fois: les titres niveau 1 => niveau 2 et ceux de niveau 2 => niveau 3 (pour harmoniser avec les autres versions de la leçon en EN, PT et ES). Sinon cela a l'air top!

Bien vu, merci ! Oui on va faire ça. En fait finalement il y a d'autres changements à apporter, pour qu'il y ait une cohérence entre la partie 1 et la partie 2 (qui est en cours d'évaluation). Célian (le traducteur) s'en occupe la semaine prochaine !

datalogism commented:
Hello @marie-flesch  c'est tout bon pour moi :)
Bien repris ton commentaire @spapastamkou, j'ai aussi corrigé cela sur la partie 2 !

spapastamkou committed:
Merge branch 'gh-pages' into Publish-Du-HTML-a-une-liste-de-mots-(partie-1)

anisa-hawes committed:
Update du-html-a-une-liste-de-mots-1.md

Apply changes from https://github.com/programminghistorian/ph-submissions/commit/bbc78e94f3ea23ddb150a55fa345f944faf79a60

anisa-hawes committed:
Update du-html-a-une-liste-de-mots-1.md

Update `translation_date:` 
Update `translation-reviewer:` (singular from plural)

Commit:
Merge branch 'gh-pages' into Publish-Du-HTML-a-une-liste-de-mots-(partie-1)

anisa-hawes commented:
Hello @marie-flesch. I've integrated Célian's most recent edits https://github.com/programminghistorian/ph-submissions/commit/bbc78e94f3ea23ddb150a55fa345f944faf79a60 and brought this branch up-to-date with gh-pages. 

I'm unclear about how to mark the changes requested by Sofia as resolved so that we can merge... I'll investigate, and update you as soon as possible.

marie-flesch commented:
Thank you so much @anisa-hawes, it looks good!

spapastamkou commented:
@anisa-hawes resolved :-) - thanks!
